.github/ISSUE_TEMPLATE/suggest_wallet.md

```diff
@@ -6,98 +6,192 @@ labels: "wallet :purse:,"
 assignees: ""
 ---
 
-_Please note as part of [our Q2 product roadmap](https://github.com/ethereum/ethereum-org-website/issues/6161) we paused adding new wallets until we establish new, expanded criteria._
+Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/). Only continue with the issue if your wallet meets the criteria listed there.
 
-Before suggesting a wallet, make sure you've read [our listing policy](https://www.ethereum.org/en/contributing/adding-products/).
+If the wallet does meet our list policy, please complete the following information which we need to accurately list the wallet.
 
-Only continue with the issue if your wallet meets the criteria listed there.
+### Project info
 
-If it does complete the following information which we need to accurately list the wallet.
+**Wallet name**
 
-**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team or some other method.**
+  <!-- Please provide the official name of the wallet -->
+
+**Wallet description**
+
+<!-- Please provide a short 1-2 sentence description of the wallet without marketing claims. Avoid claims like “the best Ethereum wallet”. We will ask for revisions if this comes across as marketing-focused. -->
+
+**Wallet logo**
+
+<!-- Please provide a hi-res SVG or transparent PNG -->
+
+**Background color for brand logo**
+
+<!-- Please provide a hex code for the brand color. This will be added to the background of the wallet card. -->
+
+**URL to the project**
+
+<!-- Please provide a URL (e.g. to the website of the wallet) -->
+
+**URL to the documentation**
+
+<!-- Please provide a URL to the documentation -->
+
+**Is your wallet security tested? Please explain security measures i.e. security audit, internal security team, or some other method.**
 
 <!-- Please explain any security measures you have taken to ensure your wallet is secure -->
 
 **When did your wallet go live to users?**
 
-<!-- Please provide an exact or approximate date when your wallet was usable by the public -->
+<!-- Please provide a date when your wallet was usable by the public. Please provide some user metrics for how many users are using this wallet. -->
```

Comment:
> Please provide some user metrics for how many users are using this wallet.

Not sure we need to ask for this. I could see how it'd be nice have some assurance that e.g. the wallet has has been used in production by thousands of users before encouraging all our visitors to use. On the other hand it feels like a bit of an overreach to ask for this data.

Reply:
I think any data we collect here is going to be far from accurate, and impossible to prove. I agree that it would be nice to offer some reassurances, but I don't think it's worth the ask for something like this.

In either case, I don't see anything in the current design (unless I missed it) for how this data would be used anyways. If there isn't, I would opt for removing this question.

Reply:
I agree overall. I think it would be hard to actually track this data. In an ideal world we could use this, but I agree it can be removed for now as its unlikely to be helpful.

Reply:
```suggestion
<!-- Please provide a date when your wallet was usable by the public. -->
```